index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="shortcut icon" href="img/favicon.ico" type="image/x-icon">
+    <title>Google</title>
+</head>
+
+<body>
+    <header>
+        <nav>
+          
+            <a href="">Gmail</a>
+            <a href="">Image</a>
+            <img src="img/g-menu.png" alt="">
+            <button>Sign in</button>
+       
+        </nav>
+    </header>
+    <section class="section-1">
+        <img src="img/logo.png" alt="" class="logo">
+        <form action=""> <br><br>
+            <div class="s-box">
+                <img src="img/search.svg" alt="" class="search-icon">
+                <input type="text" class="s-input">
+                <img src="img/vs.png" alt="" class="vs-icon">
+                <input type="submit" value="Google search" class="s-btn">
+                <input type="submit" value="am feeling lucky" class="s-btn">
+            </div>
+        </form>
+        <div class="lang">
+            google offers in:
+            <a href="">Francais</a>
+            <a href="">Kiswahili</a>
+            <a href="">Kinyarwanda</a>
```

Comment:
- [ ] Make sure that whenever you write an `anchor tag` all of `href` should have values, if you don't have a real link, then give a `#` as it's arbitrary value. This means instead of writing ~`a href=""`~ , you should have written `a href="#"`